packages/chips/Chip.js

```diff
@@ -35,18 +36,28 @@ export class Chip extends Component {
   componentDidMount() {
     this.foundation_ = new MDCChipFoundation(this.adapter);
     this.foundation_.init();
+    this.foundation_.setSelected(this.props.selected);
```

Comment:
Move adding/removing of the `mdc-chip--selected` class to here and `componentDidUpdate`